config/settings.py

```diff
@@ -0,0 +1,127 @@
+"""
+Django settings for config project.
+
+Generated by 'django-admin startproject' using Django 5.1.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+import os
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = "django-insecure-hxak7+phh51^p%vuxxk5%28b4tz^%e1m=6@q*tmf1eq-0$32_3"
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```

Comment:
these should come from env variables

Reply:
the values you set here should be used as default values if the env variables are not defined. Something like this:
```
DEBUG = os.environ.get("DEBUG", "True")
```

Reply:
also don't forget that every env variable is read as a string